code-challenges/map/challenges-07.test.js

```diff
@@ -0,0 +1,267 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1
+
+Write a function named forLoopTwoToThe that, given an array of integers as input, iterates over the array and returns a new array. The returned array should contain the result of raising 2 to the power of the original input element.
+
+You may choose to complete this challenge using a for loop, for...in syntax, or for...of syntax.
+
+For example, twoToThe([1,2,3]) returns [2,4,8] because 2 ^ 1 = 2, 2 ^ 2 = 4, and 2 ^ 3 = 8.
+------------------------------------------------------------------------------------------------ */
+
+const forLoopTwoToThe = (arr) => {
+  // Solution code here...
+  let myEmpArr = [];
+  for(let val of arr) {
+    myEmpArr.push(Math.pow(2, val)); 
+  }
+  return myEmpArr;
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named forEachTwoToThe that produces the same output as your forLoopTwoToThe function from challenge 1, but uses forEach instead of a for loop.
+------------------------------------------------------------------------------------------------ */
+
+const forEachTwoToThe = (arr) => {
+  // Solution code here...
+  let myEmpArr = [];
+  arr.forEach((element) => {
+    myEmpArr.push(Math.pow(2, element));
+  });
+  return myEmpArr;
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 3
+
+Write a function named mapTwoToThe that produces the same output as your forLoopTwoToThe function from challenge 1 and your forEachTwoToThe function from challenge 2, but uses map instead of a for loop or forEach.
+------------------------------------------------------------------------------------------------ */
+
+const mapTwoToThe = (arr) => {
+  // Solution code here...
+  let myEmpArr = [];
+  arr.map((val, idx) => {
```

Comment:
You included and `idx` parameter but never used it. Be sure to delete unused variables/parameters!